addons/sale/models/sale_order_line.py

```diff
@@ -266,12 +270,19 @@ class SaleOrderLine(models.Model):
 
     untaxed_amount_invoiced = fields.Monetary(
         string="Untaxed Invoiced Amount",
-        compute='_compute_untaxed_amount_invoiced',
+        compute='_compute_invoiced_amounts',
+        store=True)
+    amount_invoiced = fields.Monetary(
```

Comment:
Do we need to store this?

Reply:
Yes, because both `untaxed_amount_invoiced` and `amount_invoiced` share the same compute method, and `untaxed_amount_invoiced` was already stored (used in an SQL query in `sale_report.py`)

Reply:
> Yes, because both `untaxed_amount_invoiced` and `amount_invoiced` share the same compute method

I'll change my question then :smile: 
Does it have to be the same compute function?

Reply:
Both fields share the same dependencies and computation logic, so it made sense to use the same compute method, no?

Reply:
Well... yes. But also no.
* because we need one stored but not the other
* generally, each stored field needs needs it's own compute method

______________
Stored compute fields need to have their own computed method, otherwise
```python
class MyModel(models.Model):
    _name = 'my.model'
    fieldA = fields.Char(compute='_compute_fields', store=True, readonly=False)
    fieldB = fields.Char(compute='_compute_fields', store=True, readonly=False)

    def _compute_fields(self):
        for record in self:
            record.fieldA = 'a'
            record.fieldB = 'b'
```
```python
env['my.model'].create({'fieldA': 'c'})
# `fieldA` will be 'a' instead of 'c' because 
# `_compute_fields` needs to be called to compute `fieldB`
```